serendipity_event_markdown/serendipity_event_markdown.php

```diff
@@ -237,6 +251,49 @@ function _markdown_markup($text) {
         return Markdown($text);
     }
     */
+
+    function addToCSS(&$eventData) {
+        $eventData .= '
+/* Footnotes (generated by serendipity_event_markdown) */
+
+a.footnote-ref:after {
+  content: ")";
+}
+
+.footnotes hr {
+  border-top: dashed #cccccc;
```

Comment:
You can always write colors like `#cccccc` (6 times the same value) in their short form `#ccc`.